### DISCUSSION TITLE
大家一般玩MC哪个版本

### DISCUSSION THREAD

**Discussion by Tiger-Coder-C** [Meloong-Git/PCL#1498] [讨论]
如题

**Comment by WForst-Breeze** [comment]
个人一般会玩 最新快照、1.19.3、1.12.2 和 1.8.9 >w< 快照主要是尝尝鲜，1.19.3 一般玩的原版生存，1.12.2 是和朋友一起玩整合包，1.8.9 就随便玩玩 PVP 之类的……？~不过很菜~ 以前会玩玩 1.7.10，因为对配置要求还可以，而且优秀的整合包也很多，现在电脑上来了就基本玩高一点的版本了~
~不玩 1.19.4 的原因是我自己的整合包还没升级到 1.19.4~

**Comment by Chiloven945** [comment]
1.19.2，1.18.2，1.16.5
不太喜欢玩旧版本=。=

24/08/07 Edit
1.21，1.20.1，其余就很少使用了。

**Reply by RLlufee** [reply]
1.20.1  1.19.2  1.18.2  1.16.5  1.12.2
高版本mod  兼容mod较多（联机使用）  自己玩

**Comment by ZS2147483647** [comment]
1.19.2

**Comment by ZS2147483647** [comment]
https://www.123pan.com/s/gxXSVv-r0oxA.html
1.19.2近原版优化,一起来玩啊(本人自制,内置联机,如果用pcl2安装不上可以试试hmcl)

**Reply by Chiloven945** [reply]
内置联机？哦essential，那mod又闭源又商业化，产生依赖分分钟毁掉社区，个人不太支持……
~话说不是讨论版本怎么变成推销什么的了（~

**Comment by RawDiamondMC** [comment]
1.19.4，因为有multiconnect所以玩的都是最新版本（（（

**Reply by iPodtouch6557** [reply]
> 1.19.4，因为有multiconnect所以玩的都是最新版本（（（

~啥时候成collaborator了~所以这是个啥模组

**Reply by Chiloven945** [reply]
> > 1.19.4，因为有multiconnect所以玩的都是最新版本（（（
> 
> ~啥时候成collaborator了~所以这是个啥模组

支持高版本连接低版本的服务器

**Reply by RawDiamondMC** [reply]
> > > 1.19.4，因为有multiconnect所以玩的都是最新版本（（（
> > 
> > 
> > ~啥时候成collaborator了~所以这是个啥模组
> 
> 支持高版本连接低版本的服务器

补充：最低到1.8

**Comment by iPodtouch6557** [comment]
我主要玩1.19及以上，可能会因为地图/数据包/模组兼容问题开1.12~1.18，但是偶尔闲的没事也会挖坟远古版

**Comment by HelenW1234** [comment]
1.19.2

**Comment by xianhongtao** [comment]
主要是最新版和次新版（优化模组没开发完就次新版，其他版本用multiconnect）

**Comment by FunYun-ch1** [comment]
1.8.9起床战争或者是最新版玩生存

**Reply by iPodtouch6557** [reply]
不是，现在还有人1.8啊
不都1.12吗

**Reply by WForst-Breeze** [reply]
绝大多数旧版 PVP 服务器都停留在了 1.8.9，所以一部分玩家会选择留在 1.8.9

**Comment by MimiRabbit87** [comment]
1.11.2、1.12.2、1.16.5、1.20.1awa

**Comment by wuliaodexiaoluo** [comment]
目前在玩 1.21
因为小说群作者开了个服务器
（顺带提一嘴在服务器我被称为小洛（缩写）或者仙人 ~（在服务器明目张胆的开挂(作者默许（应该是的）)）~ ）

**Comment by kong-que-dnf123** [comment]
1.12.2和1.18.2（我主玩的两个服务器的版本号
